clang/include/clang/AST/PreorderAST.h

```diff
@@ -122,6 +122,16 @@ namespace clang {
     // @param[in] N is current node of the AST. Initial value is Root.
     void NormalizeExprsWithoutConst(Node *N);
 
+    // Get the deref offset from the DerefExpr. The offset represents the
+    // possible amount by which the bounds of an ntptr should be widened.
+    // @param[in] UpperExpr is the upper bounds expr for the ntptr.
+    // @param[in] DerefExpr is the dereferenced expr for the ntptr.
+    // @param[out] Offset is the offset from the base by which the pointer is
+    // dereferenced.
+    // return Returns a boolean indicating whether a valid offset exists.
```

Comment:
Could you clarify the meaning of true/false here?  Sometimes clang likes to reverse the meaning of booleans returned by functions, so it is helpful to be specific.